Tasks/VsTest/vstest.ts

```diff
@@ -122,6 +127,7 @@ function getVstestArguments(settingsFile: string, tiaEnabled: boolean): string[]
             if (!tl.exist(settingsFile)) {
                 // because this is filepath input build puts default path in the input. To avoid that we are checking this.
                 tl.setResult(tl.TaskResult.Failed, tl.loc('InvalidSettingsFile', settingsFile));
+                publishErrorToCi(constants.errorCodes.invalidSettingsFile, "Invalid settings file");
```

Comment:
>publishErrorToCi [](start = 16, length = 16)

Can this precede the setResult ?

Reply:
my bad

---
In reply to: [136021659](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136021659) [](ancestors = 136021659)